scripts/lsp_types.lua

```diff
@@ -17,12 +17,12 @@ local function tofile(fname, text)
 end
 
 function M.gen(opt)
-  if vim.loop.fs_stat('./lsp.json') then
+  if vim.uv.fs_stat('./lsp.json') then
     vim.fn.delete('./lsp.json')
   end
   vim.fn.system({
     'curl',
-    'https://raw.githubusercontent.com/microsoft/lsprotocol/main/generator/lsp.json',
+    'https://raw.githubusercontent.com/microsoft/lsprotocol/2023.0.0a2/generator/lsp.json',
```

Comment:
the use-case of this script is "I want to update to latest LSP spec". with this change, that now requires editing this part of the script.

this could perhaps be a parameter that defaults to `main`.